test/functional/test_framework/script_util.py

```diff
@@ -28,15 +29,18 @@
 # Output:      8 [Amount] + 1 [scriptPubKeyLen] = 9 bytes
 #
 # Hence, the scriptPubKey of the single output has to have a size of at
-# least 22 bytes, which corresponds to the size of a P2WPKH scriptPubKey.
-# The following script constant consists of a single push of 21 bytes of 'a':
-#   <PUSH_21> <21-bytes of 'a'>
-# resulting in a 22-byte size. It should be used whenever (small) fake
-# scriptPubKeys are needed, to guarantee that the minimum transaction size is
-# met.
-DUMMY_P2WPKH_SCRIPT = CScript([b'a' * 21])
-DUMMY_2_P2WPKH_SCRIPT = CScript([b'b' * 21])
-
+# least 5 bytes. DUMMY_P2WPKH_SCRIPT should be used when requiring
```

Comment:
You replaced `DUMMY_P2WPKH_SCRIPT` with `DUMMY_MIN_SPEND_SCRIPT` but didn't update this comment

But I think that doesn't make sense? The former was standard, the latter requires you to use `-acceptnonstdtxn=1`, which seems likely to hide bugs?

Reply:
Both are nonstandard. My goal was to get rid of this dummy completely by using standard scripts and/or MiniWallet, but I never got around to do it for the last remaining use here.

Reply:
Oh, it's "p2wpkh" only in the sense that it's the same size... Weird.

Reply:
These dummy constants are now being removed in #26277 along with `acceptnonstdtxn=1` usage.